workers/unity/Packages/com.improbable.gdk.transformsynchronization/Components/BufferedTransform.cs

```diff
@@ -1,16 +1,11 @@
 using Generated.Improbable.Transform;
+using Unity.Entities;
 using UnityEngine;
 
 namespace Improbable.Gdk.TransformSynchronization
 {
-    public class BufferedTransform : Component
+    public struct BufferedTransform : IBufferElementData
```

Comment:
You might need to add `[InternalBufferCapacity(somevalue)]` to indicate how many elements we expect

Reply:
added it